recipes/mpiFFT4py/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mpiFFT4py" %}
```

Comment:
Is this because of the folder name (`recipes/mpiFFT4py`)? Because I am using `package: name: {{ name|lower }}` in meta.yaml.